src/omuse/community/qgmodel/Makefile

```diff
@@ -43,4 +43,4 @@ qgmodel_worker: worker_code.f90 $(CODELIB) $(OBJS)
 	$(MPIFC) $(FCFLAGS) $(FS_FLAGS) $< $(OBJS) $(CODELIB) $(FS_LIBS) $(LDFLAGS) -o $@
 
 %.o: %.f90
-	$(MPIFC) $(FCFLAGS) -c -o $@ $<
+	$(MPIFC) $(FCFLAGS) -std=legacy -c -o $@ $<
```

Comment:
this might be better FCFLAGS+=...

Reply:
That's actually intentional in the (naive?) optimistic hope that the Fortran we generate in amuse/omuse starts complying with an actual Fortran standard so we can remove this one.

Reply:
ah ok..is there anything in the interface.f90 that is not standard? the generated worker.f90 should already be! (or at least i did some fixes there)

Reply:
I'm not sure, but it was failing with `-std=f95` and `-std=f2003` and I didn't look into that yet. I wanted to get the tests and new versioning done first, before bothering with that. The tests currently also use the PYPI version of amuse, so if there's fixing in amuse that aren't on PYPI yet, then that'd explain the difference.

Reply:
I think pypi is up to date (but @rieder might know better)

Reply:
Pypi is up to date for amuse as of yesterday